WordPress/Classes/Networking/ServiceRemoteREST.h

```diff
@@ -28,6 +34,15 @@ typedef enum
  */
 - (id)initWithApi:(WordPressComApi *)api;
 
+/**
+ *  @brief      Designated initializer.
+ *
+ *  @param      api     The API to use for communication.  Cannot be nil.
+ *
+ *  @returns    The initialized object.
+ */
+- (id)initWithComRestApi:(WordPressComRestApi *)api;
```

Comment:
Instead of adding this property here, why don't we take the opportunity and create a new base class `ServiceRemoteWordPressComREST` in Swift?